theme/src/resources/templates/header.hbs

```diff
@@ -552,16 +600,22 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 
 					// Layout Updates
 
+					const notificationHeight = siteWideNotification.offsetHeight;
+
 					const banner = document.querySelector('.homepage .banner');
 
 					if (banner) {
-						banner.setAttribute("style", "min-height: 505px;padding-top: 6rem;");
+						const newBannerStyle = 'min-height: ' + (initialBannerHeight + notificationHeight) + 'px; padding-top: ' + notificationHeight + 'px;';
+
+						banner.setAttribute("style", newBannerStyle);
```

Comment:
Can we change this to single quotes?

Reply:
On mdn, setting style attributes all had double quotes. https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style

Reply:
There was a section saying `elt.setAttribute('style', '...')`.  I think for consistency it would make more sense to use single quotes as we do the same throughout portal?  Thoughts?

Reply:
Changed on https://github.com/rdai10/zendesk/pull/236, in f9ca6b7

Reply:
Thanks @rdai10!